src/snyk/cli/services/cliService.ts

```diff
@@ -101,11 +101,15 @@ export abstract class CliService<CliResult> extends AnalysisStatusProvider {
       args.push(...parseArgsStringToArgv(additionalParams.trim()));
     }
 
+    if (!this.config.shouldReportEvents) {
+      args.push('--DISABLE_ANALYTICS');
+    }
```

Comment:
@michelkaporin, we should use `SNYK_CFG_<config-key-name>` syntax instead of command attribute. Unfortunately it's a occasion that it works with `snyk test ...` command, for other CLI commands (iac, container etc) it will fail with undefined property.

Reply:
Hmm, interesting. Can't remember why I've added switching to the CLI param as acceptance criteria of the ticket 🤔

Reply:
Updated it ✅